examples/codegen/README.md

```diff
@@ -0,0 +1,31 @@
+## End-to-end code generation example
+
+This example demonstrates an end-to-end pipeline of code generation and execution that typically occurs in a JIT engine.
+It creates a simple function (adding +1 to an array and storing the result in a second one) via llvm api,
+converts it to spirv, and submits it to the runtime.
+
+## Running the example
+
+### Using conda envrionment
+
+```
+$ conda env create -f third_party/deps.yml
+$ conda activate examples
+# configure with:
+$ cmake .. -DUR_BUILD_ADAPTER_L0=ON -DUR_BUILD_EXAMPLE_CODEGEN=ON
+$ make
+$ UR_ADAPTERS_FORCE_LOAD=./lib/libur_adapter_level_zero.so ./bin/codegen
```

Comment:
force loading the adapter shouldn't be needed.

Reply:
fixed that